src/routes/safe/components/Layout/Tabs/index.jsx

```diff
@@ -0,0 +1,135 @@
+// @flow
+import Tab from '@material-ui/core/Tab'
+import Tabs from '@material-ui/core/Tabs'
+import { withStyles } from '@material-ui/core/styles'
+import React from 'react'
+import { withRouter } from 'react-router-dom'
+
+import { styles } from './style'
+
+import {
+  ADDRESS_BOOK_TAB_BTN_TEST_ID,
+  BALANCES_TAB_BTN_TEST_ID,
+  SETTINGS_TAB_BTN_TEST_ID,
+  TRANSACTIONS_TAB_BTN_TEST_ID,
+} from '~/routes/safe/components/Layout'
+import SettingsTab from '~/routes/safe/components/Layout/Tabs/SettingsTab'
+import { AddressBookIcon } from '~/routes/safe/components/assets/AddressBookIcon'
+import { AppsIcon } from '~/routes/safe/components/assets/AppsIcon'
+import { BalancesIcon } from '~/routes/safe/components/assets/BalancesIcon'
+import { TransactionsIcon } from '~/routes/safe/components/assets/TransactionsIcon'
+
+type Props = {
+  classes: Object,
+  match: Object,
+  history: Object,
+  location: Object,
+}
+
+const TabsComponent = (props: Props) => {
+  const { classes, location, match } = props
+
+  const handleCallToRouter = (_, value) => {
+    const { history } = props
+
+    history.push(value)
+  }
+
+  const tabsValue = () => {
+    const balanceLocation = `${match.url}/balances`
+    const isInBalance = new RegExp(`^${balanceLocation}.*$`)
```

Comment:
in some other files the regex was defined in slashes `/like-that/`. Can we stick to one way? And also please rename the variable to state that this is a regex (`BALANCES_TAB_LOCATION_REGEX`)